lib/src/models/utils/mapper.dart

```diff
@@ -28,6 +32,23 @@ class QMapper {
     });
   }
 
+  static QOfferings offeringsFromJson(String? jsonString) {
```

Comment:
Just to keep the sdk in the same style I suggest using `dynamic json` instead of `String? jsonString`

Reply:
Would this not be miss leading because the other methods accept a `Map` whereas this method can only transform a String?

Or should this method cater for a Map as well and String and do a type check. Then the API would be consistent as all methods would accept a `Map<String, dynamic>` as a dynamic.

Reply:
Not my library to maintain but arguably the other methods should only accept a `Map<String, dynamic>`, imo. However, if you confirm you still want it as a dynamic, I'll change it.

Reply:
Hm, yeah, I agree, let's use `String?` like in your original version.
Other methods were written a time ago. Perhaps `dynamic` mattered in previous versions of SDK and now is redundant

Reply:
👍